### PR TITLE
display book information

### PR DESCRIPTION
**Pull request by MuhammadTamzid**
display book information

### CODE REVIEW

**Comment by sujoydatta** [comment]
ok

File: app/views/books/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+<div class="row">
+  <div class="col-md-5">
+    <p><b><%= t "title" %>:</b> <b><%= @book.title %></b></p>
+    <p><%= t "author" %>: <%= @book.author %></p>
+    <p><%= t "publish_date" %>: <%= @book.publish_date %></p>
+    <p><%= t "page" %>: <%= @book.page %></p>
+    <p><%= t "category" %>: <%= @book.category.name %></p>
+    <p>
+      <%= render "book_photo", book_photos: @book.book_photos %>
```
Comment (dieunb) [inline app/views/books/show.html.erb]: use render partial instead of it

File: app/views/books/_book_photo.html.erb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<% book_photos.each do |book_photo|%>
+  <%= image_tag book_photo.image_url(:medium).to_s %> <br><br>
```
Comment (dieunb) [inline app/views/books/_book_photo.html.erb]: i think no need `.to_s`

File: app/views/books/show.html.erb (hunk 1 above)
Comment (MuhammadTamzid) [inline app/views/books/show.html.erb]: will change

File: app/views/books/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+<div class="row">
+  <div class="col-md-5">
+    <p><b><%= t "title" %>:</b> <b><%= @book.title %></b></p>
+    <p><%= t "author" %>: <%= @book.author %></p>
+    <p><%= t "publish_date" %>: <%= @book.publish_date %></p>
+    <p><%= t "page" %>: <%= @book.page %></p>
+    <p><%= t "category" %>: <%= @book.category.name %></p>
+    <p>
+      <%= render "book_photo" %>
```
Comment (MuhammadTamzid) [inline app/views/books/show.html.erb]: Anh is it render partial?

Comment (dieunb) [inline app/views/books/show.html.erb]:
```
<%= render "book_photo" %>
```

best way

Comment (MuhammadTamzid) [inline app/views/books/show.html.erb]: OK

**Pull request merged**